play-json/shared/src/main/scala/Json.scala

```diff
@@ -6,39 +6,38 @@ package play.api.libs.json
 import java.io.InputStream
 
 /**
- * Helper functions to handle JsValues.
+ * @define jsonParam @param json the JsValue to convert
+ * @define returnStringRepr A String with the json representation
+ * @define readDescription Creates a `Reads[T]` by resolving, at compile-time, the case class fields or sealed family, and the required implicits.
+ * @define handlerTypeParam @tparam A the type for which the handler must be materialized
  */
-object Json {
-
+sealed trait JsonFacade {
```

Comment:
The question is always do the risks outweight the benefits?
I mean creating his own `JsonFacade` might be useful, so that you don't need to call `Json.using` more than once and use our own `ApplicationJson`. Tought not sure how useful that might be, it heavily depends on your second comment if we can have the `StaticBinding.methods` inside this trait.